src/test/Number.test.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import {shallow} from 'enzyme'
+
+import Number from '../components/widgets/Number'
+
+describe('Number widget component', () => {
+  const mainStat = '125'
+  const goal = '250'
+  const progress = ((mainStat / goal) * 100).toString().concat('%')
+
+  const widgetData = {
+        type: 'Number',
```

Comment:
Please do not indent like this - two space indent is all that is required.  This ends up being a *huge* maintenance cost if we add attributes later.